src/app/auth/auth.service.ts

```diff
@@ -0,0 +1,39 @@
+import { Injectable } from '@angular/core';
+
+import { IUser, User } from '../shared/user.model';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class AuthService {
+  constructor() {}
+
+  public login(user): IUser {
+    const userData = {
+      id: user.email,
+      firstName: user.email,
+      lastName: user.email,
+    };
+
+    localStorage.setItem('user', JSON.stringify(userData));
```

Comment:
I don't like direct usage of browser API. All the things you use from there should be imported from some wrappers.
In classic Angular way, LS should be provided by a wrapper-service either implemented by yourself or somebody else.
Main point of it is testability and polymorphism because this service is reusable/shared across the whole app service.